detection_rules/main.py

```diff
@@ -113,8 +113,8 @@ def name_to_filename(name):
 
 
 @root.command('toml-lint')
-@click.option('--rule-file', '-f', type=click.Path(), multiple=True,
-              help='Optionally specify rule files')
+@click.option('--rule-file', '-f', multiple=True, type=click.Path('r'),
```

Comment:
@brokensound77 this look right?
```suggestion
@click.option('--rule-file', '-f', multiple=True, type=click.Path(exists=True),
```

Reply:
👍